.gitignore

```diff
@@ -0,0 +1,4 @@
+**/.DS_Store
+**/node_modules
+**/build
+**/package-lock.json
```

Comment:
New line at the end of file is a good practice :)

Reply:
Yes...I will, Thank You @paulinadubas for your time